bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_VBF_mNgtmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_VBF_V-0p01_e_NLO_run_card.dat

```diff
@@ -0,0 +1,167 @@
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1	=  run_tag  ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1000	=  nevents  ! Number of unweighted events requested 
+  0.001	=  req_acc  ! Required accuracy (-1=auto determined from nevents)
+  -1	=  nevt_job ! Max number of events per job in event generation. 
+                 !  (-1= no split).
+#***********************************************************************
+# Normalize the weights of LHE events such that they sum or average to *
+# the total cross section                                              *
+#***********************************************************************
+  average	=  event_norm       ! average or sum
+#***********************************************************************
+# Number of points per itegration channel (ignored for aMC@NLO runs)   *
+#***********************************************************************
+  0.01	=  req_acc_FO        ! Required accuracy (-1=ignored, and use the 
+ 	                   ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+  5000	=  npoints_FO_grid   ! number of points to setup grids
+  4	=  niters_FO_grid    ! number of iter. to setup grids
+  10000	=  npoints_FO        ! number of points to compute Xsec
+  6	=  niters_FO         ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+  0	=  iseed        ! rnd seed (0=assigned automatically=default))
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+  1	=  lpp1     ! beam 1 type (0 = no PDF)
+  1	=  lpp2     ! beam 2 type (0 = no PDF)
+  6500.0	=  ebeam1   ! beam 1 energy in GeV
+  6500.0	=  ebeam2   ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+'lhapdf'    = pdlabel     ! PDF set
+ 325100=lhaid
```

Comment:
Please make sure you have the systematic variation of 325100 in runcmsgrid.sh. Otherwise there will be no chance to evaluate PDF systematics.